bin/pana.dart

```diff
@@ -106,15 +99,24 @@ main(List<String> args) async {
   try {
     try {
       var analyzer = await PackageAnalyzer.create(pubCacheDir: tempPath);
-      var summary = await analyzer.inspectPackage(pkg,
-          version: version, pubHostedUrl: result['hosted-url']);
 
+      final target = result.rest.first;
+      final isLocalDir = await FileSystemEntity.isDirectory(target);
```

Comment:
Uh...not ideal. there is a package `test` for example.

I'd vote for `--source` with `site` (default) and `directory` as options.

Reply:
good point. I'm not sure why you would need `site`, I've added `--source <directory>`, PTAL

Reply:
Ah! See below for pub global activate

<package> is usually the package name (from hosted).

--source let's you say "no, <package> is a directory", etc.

Would be nice to align w/ what pub does here anyway...

```
Usage: pub global activate <package...>
-h, --help              Print this usage information.
-s, --source            The source used to find the package.
                        [git, hosted (default), path]

-f, --features          Feature(s) to enable.
-F, --omit-features     Feature(s) to disable.
    --no-executables    Do not put executables on PATH.
-x, --executable        Executable(s) to place on PATH.
    --overwrite         Overwrite executables from other packages with the same name.

Run "pub help" to see global options.
```

Reply:
How about now?